src/routes.js

```diff
@@ -51,6 +51,9 @@ const router = async app => {
 	// Get an express router from the Probot application instance
 	const router = app.route("/tako");
 
+	// Note: While Tako is starting up, /__gtg responds with a 404 status code.
+	router.get("/__gtg", (req, res) => res.send("OK"));
```

Comment:
This was `/__gtg` but now it's `/tako/__gtg`, which I argue is more correct, because Probot.